gui_agents/s1/aci/LinuxOSACI.py

```
@@ -669,7 +669,7 @@ def _create_atspi_node(
         ]:
             try:
                 attribute_dict[f"{value_key}{attr_name}"] = str(attr_func())
-            except:
+            except Exception:
                 pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)